devops/auto_node/scripts/auto_node.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+container_name="harmony_node"
+
+if [ "$EUID" = 0 ]
+  then echo "Do not run as root, exiting..."
+  exit
+fi
+
+case "${1}" in
+  "run")
+    if [ "$(docker inspect -f '{{.State.Running}}' $container_name)" = "true" ]; then
+      docker kill "${container_name}"
+    fi
+    if [ "$(docker ps -a | grep $container_name)" ]; then
+      docker rm "${container_name}"
+    fi
+    rm -rf ./."${container_name}"
+    mkdir ./."${container_name}"
+    echo ""
+    echo "[!] Storing node files on host machine at: $(pwd)/.${container_name}"
+    echo ""
+    eval docker run --name "${container_name}" -v "$(pwd)/.${container_name}:/root/node" -it danielharmony/sentry "${@:2}"
```

Comment:
how did you update danielharmony/sentry image? do you have a jenkins job?

Reply:
I changed the docker image to harmonyone/sentry. 
But one does not need to update the docker image since, on execution, it downloads the latest node binary and CLI.